core/src/main/java/ar/com/wolox/wolmo/core/extensions/GenericsExtensions.kt

```diff
@@ -0,0 +1,6 @@
+package ar.com.wolox.wolmo.core.extensions
+
+fun <T> T?.orElse(other: T) = this ?: other
+
+val <T> T?.unit
```

Comment:
Why I would like to receive an `Unit` type? I don't understand the aim of this extension

Reply:
Oooh, right! I wanted to discuss it before and I forgot. We'll discuss it in private 😄.